Client.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,6 +2,6 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "self:Client.xcodeproj">
+      location = "self:">
```

Comment:
```suggestion
      location = "self:Client.xcodeproj">
```
This file should technically be added to gitignore but since its already added lets revert the changes